apps/pie-docs/src/patterns/cookie-banner/code/code.md

```diff
@@ -111,12 +102,12 @@ Use of locales in Vue:
 // Vue templates (using Nuxt 3)
 <script setup lang="ts">
   import '@justeattakeaway/pie-cookie-banner';
-  import enGB from "@justeattakeaway/pie-cookie-banner/locales/en-gb.json";
 </script>
 
 <pie-cookie-banner
   hasPrimaryActionsOnly
-  :locale='enGB'
+  :langauge='en'
+  :tenant='GB'
```

Comment:
As we don't have the concept of "tenant" yet in the design system, is it worth assessing the naming at this point? I think country would be easier to understand and seems less idiomatic.

Reply:
Yes I started down the Tenant path and then stuck with it but I will change to Country